nanoFirmwareFlasher/FirmwarePackage.cs

```diff
@@ -97,7 +97,7 @@ static FirmwarePackage()
 
             // NOTE: the query seems to be the oposite, it should be LESS THAN.
             // this has been reported to Cloudsmith and it's being checked. Maybe need to revisit this if changes are made in their API.
-            string requestUri = $"{repoName}/?page_size=500&q=uploaded:'>1 month ago' AND tag:{platform}";
+            string requestUri = $"{repoName}/?page_size=500&q=uploaded:'>{(preview ? "1" : "6")} month ago'{(platform.HasValue ? "AND tag:" + platform.Value : "")}"; // AND tag:{platform}";
```

Comment:
That's it!
Please add a comment with the explanation on the "magic" numbers 1 and 6. 😉 
OK to remove to the last comment with the AND

Reply:
Oops, Haste makes waste! Fixed